packages/devtools_app/lib/src/shared/ui/search.dart

```diff
@@ -970,7 +980,8 @@ class StatelessSearchField<T extends SearchableDataMixin>
     this.suffix,
     this.style,
     this.searchFieldHeight,
-  });
+    bool shouldExpandHeight = false,
+  }) : _shouldExpandHeight = shouldExpandHeight;
```

Comment:
same question here: can we just make this public so that we can use `this.shouldExpandHeight = false` in the constructor?

Reply:
see https://github.com/flutter/devtools/pull/8855#discussion_r1943768708 before addressing this comment.

Reply:
I love it. Done.